packages/gatsby-source-drupal/src/normalize.js

```diff
@@ -23,6 +29,15 @@ const nodeFromData = (datum, createNodeId) => {
 
 exports.nodeFromData = nodeFromData
 
+const isEntityReferenceRevision = type => type.indexOf(`paragraph`) === 0
```

Comment:
This all looks good to me. My only question is whether we will need to support other entity revision types besides paragraphs?

It might be difficult to know what the entity type would be though but perhaps we could make it configurable so they could add additional entity reference types to check inside their gatsby-config.js.

I don't know that we need that now, but I wanted to bring it up to see if you thought about it. It's a feature we could add later if it is requested.

Reply:
I've added configuration support and some docs to go with it

https://github.com/gatsbyjs/gatsby/pull/29289/files#diff-a86baf1b4c6ca83c03f3142e9e29272a081e2d35c2779db615b66b4845a293c6R269